packages/grpc-native-core/index.d.ts

```diff
@@ -983,14 +989,14 @@ declare module "grpc" {
      * @param options Options map
      * @return An event emitter for stream related events
      */
-    makeServerStreamRequest(
+    makeServerStreamRequest<R>(
```

Comment:
The same concerns about conflating the request and response types apply here too.